container/libcontainer/compatibility.go

```diff
@@ -178,8 +178,12 @@ type v1State struct {
 	Config v1Config `json:"config"`
 }
 
-// Relative path to the libcontainer execdriver directory.
-const libcontainerExecDriverPath = "execdriver/native"
+const (
+	// Relative path to the libcontainer execdriver directory.
+	libcontainerExecDriverPath = "execdriver/native"
+	// Absolute path of the containerd directory.
+	containerdPath = "/run/containerd"
```

Comment:
What information do we need from the config? Just curious. I can check the code of course :)

BTW, I think we can check the version of docker to decide where to look for the config, cant' we?